src/shared/types.ts

```diff
@@ -103,9 +109,11 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_EN_INTL',
     ianaTimezone: 'Asia/Kolkata',
     prospectTypes: [
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
-      ProspectType.APPROVED,
+      ProspectType.COUNTS,
+      ProspectType.TIMESPENT,
+      ProspectType.RECOMMENDED,
+      ProspectType.TITLE_URL_MODELED,
```

Comment:
url modeled is not included in this doc: 
![Screen Shot 2023-05-02 at 10 59 38 AM](https://user-images.githubusercontent.com/12415584/235747176-49a99873-4654-4460-abb6-9ca354bc85d8.png)
should we be removing this?

Reply:
yep, incorrectly added. Will remove, thanks!